src/Akeneo/Pim/Enrichment/Component/Product/Model/ProductModelInterface.php

```diff
@@ -135,4 +135,9 @@ public function getLabel(?string $localeCode, ?string $scopeCode): string;
      * @return ValueInterface|null
      */
     public function getImage(): ?ValueInterface;
+
+    /**
+     * Return the categories for the current level
+     */
+    public function getCategoriesForThisLevel(): Collection;
```

Comment:
```suggestion
    public function getCategoriesForCurrentLevel(): Collection;
```
`This` might be ambiguous